src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -76,6 +76,12 @@ private static TracerProviderBuilder SetExporter(this TracerProviderBuilder buil
                     });
                     break;
                 case "otlp":
+#if NETCOREAPP3_1
+                    // Adding the OtlpExporter creates a GrpcChannel.
+                    // This switch must be set before creating a GrpcChannel/HttpClient when calling an insecure gRPC service.
+                    // See: https://docs.microsoft.com/aspnet/core/grpc/troubleshoot#call-insecure-grpc-services-with-net-core-client
+                    AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
```

Comment:
We should be careful with setting a switch like this in a customer app. Once this is no longer a proof of concept, we should probably consider putting something like this behind a config setting.

Reply:
Issue created https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/217